rhobot.go

```diff
@@ -201,8 +201,13 @@ func main() {
 }
 
 func healthcheckRunner(config *config.Config, healthcheckPath string, reportPath string, emailListPath string) {
-	healthChecks := healthcheck.ReadYamlFromFile(healthcheckPath)
+	healthChecks, err := healthcheck.ReadHealthCheckYAMLFromFile(healthcheckPath)
+	if err != nil {
+		log.Fatal("Failed to read healthchecks!")
```

Comment:
Or you could include the err message in the Fatal log message like you did above, I had them as separate lines to indicate that 1) an error happened, and what it was, then 2) Since there was an Error in required function, a separate fatal message is thrown to indicate that Rhobot can't continue.  I'm open to either way though.